src/coffee/mapping-utils/csv.coffee

```diff
@@ -39,9 +39,15 @@ class CsvMapping
 
   _getValue: (order, mapping) ->
     value = access order, mapping[0]
-    return '' unless value
+    if not value and value != 0
+      return ''
+
     if _.size(mapping) is 2 and _.isFunction mapping[1]
       mapping[1].call undefined, value
+    else if isMoneyFormat value
+      formatMoney value
+    else if _.isNumber value
+      value
     else
       value or ''
```

Comment:
If we already check on line 42 that `value` is neither falsy nor 0, and we handle the case where it's 0, it means `value` should never be falsy on this line

Reply:
Redundant code removed here 5cfa2a7